src/main/java/chess/dao/springjdbc/SpringGameDao.java

```diff
@@ -34,33 +35,52 @@ public void update(ChessGameDto dto) {
     }
 
     @Override
-    public ChessGameDto findById(int id) {
-        String sql = "SELECT id, name, status, turn FROM game WHERE id = ?";
+    public ChessGameDto findById(Long id) {
+        String sql = "SELECT id, name, status, turn, password FROM game WHERE id = ?";
         return jdbcTemplate.queryForObject(sql, chessGameDtoRowMapper, id);
     }
 
     @Override
-    public void updateStatus(StatusDto statusDto, int id) {
+    public void updateStatus(StatusDto statusDto, Long id) {
         String sql = "UPDATE game SET status = ? WHERE id = ?";
         jdbcTemplate.update(sql, statusDto.getStatus(), id);
     }
 
     @Override
     public GamesDto findAll() {
-        String sql = "SELECT id, name, status, turn FROM game";
+        String sql = "SELECT id, name, status, turn, password FROM game";
         return new GamesDto(jdbcTemplate.query(sql, chessGameDtoRowMapper));
     }
 
     @Override
-    public int createGame(String name) {
-        String sql = "INSERT INTO game SET name = ?";
+    public Long createGame(String name, String password) {
+        String sql = "INSERT INTO game(name, password) VALUES (?, ?)";
         KeyHolder keyHolder = new GeneratedKeyHolder();
         jdbcTemplate.update(con -> {
             PreparedStatement prepareStatement = con.prepareStatement(sql, new String[]{"id"});
             prepareStatement.setString(1, name);
+            prepareStatement.setString(2, password);
             return prepareStatement;
         }, keyHolder);
 
-        return Objects.requireNonNull(keyHolder.getKey()).intValue();
+        return Objects.requireNonNull(keyHolder.getKey()).longValue();
+    }
+
+    @Override
+    public void deleteGame(Long id) {
+        String sql = "DELETE FROM game WHERE id = ?";
+        jdbcTemplate.update(sql, id);
+    }
+
+    @Override
+    public boolean existsById(Long gameId) {
+        String sql = "SELECT COUNT(*) FROM game where id = ?";
+        return jdbcTemplate.queryForObject(sql, Long.class, gameId) == 1;
```

Comment:
해당 라인은 전혀 문제가 되는 건 아니지만

```suggestion
        return jdbcTemplate.queryForObject(sql, Long.class, gameId) != 0;
```

0이 아닌지 여부로 확인하는 것도 좋은 것 같아서 의견 남겨둬봤어요!! :)
사실 '하나가 존재한다'가 전혀 이상한 것은 아니지만 '0이 아니다'가 exists를 의미하기에 더 범용적인 의미라고 생각이  되어서요!! 존재한다는 한 개 이상 존재한다도 의미가 될 수 있으니깐 0개가 아니다가 좀 더 자연스럽지 않을까? 라고 생각했어요 😄

Reply:
`하나가 존재한다` 라는 말보다 명시적으로 `0개가 아니다` 라는게 네이밍으로 보나 더 의미전달이 명확한 것 같습니다!